src/scripts/migrate.py

```diff
@@ -0,0 +1,143 @@
+"""Slurp migration pipeline
+
+Basically, we:
+1. Look at all unmapped terms T
+2. If all of parents of T are mapped, designate for slurping (we only slurp if parents are already slurped, iteratively)
+3. Extract basic information about T and export as ROBOT template
+
+Resources
+- https://incatools.github.io/ontology-access-kit/
+- https://incatools.github.io/ontology-access-kit/intro/tutorial02.html
+
+TODO's:
+  -
+"""
+import os
+import pickle
+from argparse import ArgumentParser
+from typing import Dict, List, Set, Union
+
+import pandas as pd
+import yaml
+from oaklib.implementations import ProntoImplementation
+from oaklib.resource import OntologyResource
+from oaklib.types import CURIE, URI
+
+from migrate_utils import CACHE_DIR, PREFIX, Term, _get_all_owned_terms, _get_next_available_mondo_id, \
```

Comment:
FYI: `migrate_utils.py`: I moved a lot of code into `migrate_utils`. This is basically all the stuff that allows us to write simpler code using a Python API that would be convenient. I think that when we have time, we might want to review this and see what we think we should actually suggest as changes to be incorporated into OAK, if possible.